sdk/core/core-amqp/src/retry.ts

```diff
@@ -130,14 +156,17 @@ export async function retry<T>(config: RetryConfig<T>): Promise<T> {
   if (config.delayInSeconds == undefined || config.delayInSeconds < 0) {
     config.delayInSeconds = defaultDelayBetweenRetriesInSeconds;
   }
+  if (config.operationTimeoutInMs == undefined) {
+    config.operationTimeoutInMs = defaultOperationTimeoutInSeconds * 1000;
```

Comment:
Yes, also - looks like some operations already use this value individually. So any conversions that are already happening may need to be updated as well.

@ramya-rao-a Regarding overall comment about approach, if individual operations have implemented timeout locally then is this issue only to review and update the timeout values used in all of the clients locally? Then this would not have anything to do with retry at all correct?

I thought this was related to retry and we wanted to simplify by having the operation execution and timeout handling take place in one place core-amqp.
Also, I think this has benefit in reducing code duplication to have it here and have clients pass in the operation specific value to core-amqp for use on the executions. Thoughts?

Reply:
@ramya-rao-a please share pros cons of having changes out of `core-amqp`. That would make it easier for me to reason, respond to why it might or might not be preferable to have it in here.